UnityProject/Assets/Scripts/Objects/Research/Artifacts/Effects/TelepaticArtifactEffect.cs

```diff
@@ -34,21 +36,16 @@ public override void DoEffectPulse(GameObject pulseSource)
 
 	private void IndocrinateMessageArea()
 	{
-		// get effect shape around artifact
 		var objCenter = gameObject.AssumedWorldPosServer().RoundToInt();
-		var shape = EffectShape.CreateEffectShape(EffectShapeType.Square, objCenter, auraRadius);
-
-		foreach (var pos in shape)
+		var xMin = objCenter.x - auraRadius;
+		var yMin = objCenter.y - auraRadius;
+		var bounds = new BoundsInt(xMin, yMin, 0, 20, 20, 1);
+		foreach (var connected in PlayerList.Instance.InGamePlayers)
```

Comment:
think you can use physics2D circle overlap tho  I do want to convert that to a custom solution may be some time (probably won't happen)

Reply:
hitMask = LayerMask.GetMask( "Players")
var player = Physics2D.OverlapCircleAll(transform.position, 20f, hitMask);

Reply:
Good point, thanks.